docs/contributing/documentation-guidelines/index.md

```diff
@@ -13,9 +15,46 @@ Examples of larger changes include:
 - Adding new pages with a large amount of detail, such as a tutorial
 - Re-organization of the existing documentation structure
 
-In terms of style, you should refer to the [Google developer documentation style guide](https://developers.google.com/style) as much as possible. Reading the [Highlights page](https://developers.google.com/style/highlights) of that guide is recommended, but if not then the key points below should be noted.
+## Style guide
+
+You should refer to the [Google developer documentation style guide](https://developers.google.com/style) as much as possible. Reading the [Highlights page](https://developers.google.com/style/highlights) of that guide is recommended, but if not then the key points below should be noted.
 
 - [Use standard American English spelling](https://developers.google.com/style/spelling) and punctuation.
 - [Use sentence case](https://developers.google.com/style/capitalization) for document titles and section headings.
 - [Use descriptive link text](https://developers.google.com/style/link-text).
 - [Write short sentences](https://developers.google.com/style/translation#write-short,-clear,-and-precise-sentences) that are easy to understand and translate.
+
+## Tips
+
+### How to preview your modification
+
+There are two ways to preview your modification on a documentation website.
+
+#### 1. Using GitHub Actions workflow
+
+Follow the steps below.
+
+1. Create a pull request to the repository.
+2. Add the `documentation` label from the sidebar (See below figure).
+3. Wait for a couple of minutes, and the `github-actions` bot will notify the URL for the pull request's preview.
+
+![documenatation_label](images/documentation-label-for-pull-request.png){ width="800" }
+
+#### 2. Running an MkDocs server in your local environment
+
+Instead of creating a PR, you can use the `mkdocs` command to build Autoware's documentation websites on your local computer.
+Assuming that you are using Ubuntu OS, run the following to install the required libraries.
+
+```bash
+python3 -m pip install -U $(curl -fsSL \
+  https://raw.githubusercontent.com/autowarefoundation/autoware-github-actions/main/deploy-docs/mkdocs-requirements.txt)
```

Comment:
[nits] Do you need a `\` here? If you want to make it multiline, I feel it's better to add a `\` before `$(curl`.

```suggestion
python3 -m pip install -U $(curl -fsSL 
 https://raw.githubusercontent.com/autowarefoundation/autoware-github-actions/main/deploy-docs/mkdocs-requirements.txt)
```

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/203/commits/88236e298c7fe5349de0703041d0a8630f83872f
Just removed the backslash